lwt/nbd_lwt_unix.ml

```diff
@@ -21,13 +21,9 @@ type tls_role =
   | TlsClient of Ssl.context
   | TlsServer of Ssl.context
 
-let debug_io = ref false
-
-(* TODO move this function into a library used by this and other executables *)
+(* TODO move this function into a library used by both this and vhd-tool. *)
 (* Also the read/write/close functions that are inside tls_channel_of_fd *)
-let io_complete name offset op fd buffer =
-  if !debug_io
-  then Printf.fprintf stderr "%s offset=%s length=%d\n%!" name (match offset with Some x -> Int64.to_string x | None -> "None") (Cstruct.len buffer);
+let io_complete op fd buffer =
   let ofs = buffer.Cstruct.off in
   let len = buffer.Cstruct.len in
   let buf = buffer.Cstruct.buffer in
```

Comment:
Even more confusing things about this function:
* The `ofs` parameter of `loop` is not used at all. We just fill the whole buffer starting from offset 0.
* Given the above, `len` could be renamed to `remaining_bytes`, and `acc` could be renamed to something like `processed_bytes`
* This function could benefit from labelled parameters, so we can see from the invocations what goes where
* The `fd` and `buf` parameters of `loop` are unnecessary - they are the same in this whole function
I think these changes will make this function much more readable and easier to understand.